app/v2/services/MessagesXmlParsingService.scala

```diff
@@ -74,7 +74,7 @@ class MessagesXmlParsingServiceImpl @Inject() (implicit materializer: Materializ
         .toMat(messageTypeSink)(Keep.right)
         .run()
         .recover {
-          case NonFatal(e) => Left(ExtractionError.MalformedInput())
+          case NonFatal(_) => Left(ExtractionError.MalformedInput)
```

Comment:
In the tests, can we verify this ExtractionError gets returned when a malformed input is supplied?

Reply:
Do we not have that? This was just an adjustment to a previous PR to make a case class a case object, as it took no inputs, so there should be tests for that already.

Reply:
Don't think there are though, should've been done as part of the previous PR but thought I'd mention here since I couldn't see any

Reply:
I shall take a look!